pydynamo_brain/pydynamo_brain/ui/topMenu.py

```diff
@@ -79,6 +80,10 @@ def __init__(self, stackWindow):
             self.alignIDsToNew, QtCore.Qt.SHIFT + QtCore.Qt.Key_Apostrophe))
         menuBar.addMenu(editMenu)
 
+        manualRadiiSubmenu = editMenu.addMenu("Radii Mode")
+        radii(manualRadiiSubmenu.addAction("Recursive Radii Estimation", self.radiiEstimator, QtCore.Qt.SHIFT + QtCore.Qt.Key_G))
```

Comment:
I'd go with single first, then recursive next.
To align with marking, maybe also names like:
"Estimate selected point radius"
"Estimate radius of all downstream points"